docs/unified-test-documentation/dasharo-compatibility/323-docking-station-detect.md

```diff
@@ -0,0 +1,139 @@
+# Dasharo compatibility: Docking station detect
+
+## Test cases
+
+### Common
+
+**Test setup**
+
+1. Proceed with the
+    [Generic test setup: firmware](../../generic-test-setup/#firmware).
+1. Proceed with the
+    [Generic test setup: OS installer](../../generic-test-setup/#os-installer).
+1. Proceed with the
+    [Generic test setup: OS installation](../../generic-test-setup/#os-installation).
+1. Proceed with the
+    [Generic test setup: OS boot from disk](../../generic-test-setup/#os-boot-from-disk).
+1. The docking station connected to the USB-C port.
+
+### DUD001.001 USB detection after coldboot (Ubuntu 22.04)
```

Comment:
Incorrect test case name. In this should be smth like this: `Docking station detection after coldboot (Ubuntu 22.04)`